.github/workflows/check_make_vtadmin_web_proto.yml

```diff
@@ -0,0 +1,48 @@
+name: check_make_vtadmin_web_proto
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Check Make VTAdmin Web Proto
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Check for changes in relevant files
+      uses: frouioui/paths-filter@main
+      id: changes
+      with:
+        token: ''
+        filters: |
+          proto_changes:
```

Comment:
It might be worth adding `web/vtadmin/src/proto` to this list, to catch potentially weird scenarios where the .js is updated but the .proto is not.

Reply:
excellent point, added, thanks!